src/db/cards.db.js

```diff
@@ -0,0 +1,75 @@
+let status = ['inqueue', 'inprogress', 'done'];
```

Comment:
A better way to structure constants would be:
```suggestion
const IN_QUEUE = 'inqueue';
const IN_PROGRESS = 'inprogress';
const DONE = 'done';
```
And down below when you reference these constants, it's better to use variables since it's more descriptive rather than array positions.